modules/apps/frontend-js/frontend-js-react-web/src/main/resources/META-INF/resources/js/hooks/useLiferayState.ts

```diff
@@ -12,7 +12,7 @@
  * details.
  */
 
-import {State} from '@liferay/frontend-js-state-web';
+//import {State} from '@liferay/frontend-js-state-web';
```

Comment:
At what point would we be able to actually import this instead of commenting out? When this is uncommented, will the build time just increase back to what we are used to?

Reply:
This is addressed by the last pushed commit :+1:

Reply:
> will the build time just increase back to what we are used to?
Nope, because it is an external, so webpack stops there and simply renders an `import from ...` in the resulting .js file.